pkg/controller/workspace/config/config.go

```diff
@@ -1,6 +1,6 @@
 // MY LICENSEdep
```

Comment:
I know this isn't part of the changes in this PR, but this looks strange.

Reply:
This was probably created when the project files were initially bootstrapped by the `operator-sdk`. Afair there was an option to provide a license header, which I didn't do at first, which produced this sort-of default license comment.

We should probably just choose a license and add the license comments manually in files.

Reply:
:+1: I think some other files already have a license header.

Reply:
Added Eclipse Public License - v 2.0 - the same as we have for Che Plugin Broker
https://github.com/che-incubator/che-workspace-crd-operator/pull/3/commits/33352e3188079cd86ec8071ed4da64664bdbb7d5

Let me know if someone has another suggestion